build.gradle.kts

```diff
@@ -181,21 +197,29 @@ tasks {
     }
   }
 
-  downloadRobotServerPlugin {
-    version.set(remoteRobotVersion)
-  }
+  // TODO: Robot server task not yet available
+//  downloadRobotServerPlugin {
+//    version.set(remoteRobotVersion)
+//  }
 
-  runIdeForUiTests {
-    systemProperty("robot-server.port", "8082")
-    systemProperty("ide.mac.message.dialogs.as.sheets", "false")
-    systemProperty("jb.privacy.policy.text", "<!--999.999-->")
-    systemProperty("jb.consents.confirmation.enabled", "false")
-    systemProperty("ide.show.tips.on.startup.default.value", "false")
-    systemProperty("octopus.handler", System.getProperty("octopus.handler") ?: true)
-  }
+  // TODO: Use testIdeUi task
+//  runIdeForUiTests {
+//    systemProperty("robot-server.port", "8082")
+//    systemProperty("ide.mac.message.dialogs.as.sheets", "false")
+//    systemProperty("jb.privacy.policy.text", "<!--999.999-->")
+//    systemProperty("jb.consents.confirmation.enabled", "false")
+//    systemProperty("ide.show.tips.on.startup.default.value", "false")
+//    systemProperty("octopus.handler", System.getProperty("octopus.handler") ?: true)
+//  }
```

Comment:
Also this task? I don't know how the current UI tests work...

Reply:
WIP, tasks will be available soon.